modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -198,18 +198,22 @@ public void validate() {
     getZookeeperTimeout();
   }
 
-  public FluoConfiguration setApplicationName(String applicationName) {
+  public FluoConfiguration setFluoApplicationName(String applicationName) {
     verifyApplicationName(applicationName);
     setProperty(CLIENT_APPLICATION_NAME_PROP, applicationName);
     return this;
   }
 
-  public String getApplicationName() {
+  public String getFluoApplicationName() {
     String applicationName = getString(CLIENT_APPLICATION_NAME_PROP);
     verifyApplicationName(applicationName);
     return applicationName;
   }
 
+  public String getYarnApplicationName() {
```

Comment:
Thinking this method should not be in API config class.   There is no way to set it. Also, we are trying to avoid any requirements to use yarn.  Thinking this should be moved out of API to some implementation package.